charts/router/values.yaml

```diff
@@ -5,3 +5,13 @@ platform_domain: ""
 dhparam: ""
 # limits_cpu: "100m"
 # limits_memory: "50Mi"
+
+# Any custom router annotations(https://github.com/deis/router#annotations)
+# which needs to be applied can be specified as key-value pairs under deployment_annotations key
```

Comment:
Minor tweaks:
"needs" -> "need"

Either say `the "deployment_annotations" key` or just `"deployment_annotations"`